plugins/bcli.c

```diff
@@ -660,32 +660,43 @@ static struct command_result *process_getrawblock(struct bitcoin_cli *bcli)
 		plugin_log(bcli->cmd->plugin, LOG_DBG,
 			   "failed to fetch block %s from the bitcoin backend (maybe pruned).",
 			   stash->block_hash);
-		
-		if (!stash->peers) {
-			/* We don't have peers to fetch blocks from, get some! */
-			start_bitcoin_cli(NULL, bcli->cmd, process_getpeerinfo, true,
-					  BITCOIND_HIGH_PRIO, stash,
-					  "getpeerinfo", NULL);
-			
-			return command_still_pending(bcli->cmd);
-		}
-
-		if (tal_count(stash->peers) > 0) {
-			/* We have peers left that we can ask for the block */
-			start_bitcoin_cli(
-			    NULL, bcli->cmd, process_getblockfrompeer, true,
-			    BITCOIND_HIGH_PRIO, stash, "getblockfrompeer",
-			    stash->block_hash,
-			    take(tal_fmt(NULL, "%i", stash->peers[0])), NULL);
 
-			return command_still_pending(bcli->cmd);
+		if (bitcoind->version >= 230000) {
```

Comment:
Shouldn't we just increase our minimum version? Even 23 is EOL...

Reply:
Would it make sense to directly jump to 25 which is the oldest that still gets maintenance updates?

Reply:
Mh I think we need to consider `elementsd` versions as well. Maybe this is why min version was set to 22.

Reply:
I updated `elementsd` on CI to `23.2.1` for now 5ab1ff5. This fixes the liquid-cln tests. It is a bit weird that the `bcli` backend utilizes `bitcoind` as well as `elementsd` while their versions are drifting apart.